urxvt/shellex.in

```diff
@@ -128,35 +128,6 @@ sub geometry_from_focus {
     }
 }
 
-sub slurp {
-    open my $fh, '<', shift;
-    local $/;
-    <$fh>;
-}
-
-sub gen_conf {
-    my ($cfg, $cfgname) = tempfile("/tmp/shellex-XXXXXXXX", UNLINK => 0);
-
-    print $cfg "rm $cfgname\n";
-
-    my %fileset;
-    map { $fileset{basename($_)} = 1 } <@SYSCONFDIR@/shellex/*>;
```

Comment:
looking at conf.mk, aren't the conf files in `@DESTDIR@@SYSCONFDIR@/shellex/*` ?